.github/workflows/ci-sdk.yaml

```diff
@@ -71,6 +71,8 @@ jobs:
           run: cargo +nightly udeps
         - name: Additional per-crate checks
           run: ../tools/additional-per-crate-checks.sh ./sdk/
+        - name: Failure test
+          run: exit 1
```

Comment:
I am not sure I understand why this _have_ to fail..

Reply:
I'm just testing failure with that 😄 Will remove it once I know it works.

Reply:
Ah! Oook 😄